packages/flutter/lib/src/material/radio.dart

```diff
@@ -129,6 +132,7 @@ class Radio<T> extends StatefulWidget {
     this.visualDensity,
     this.focusNode,
     this.autofocus = false,
+    this.useAdaptiveCheckmarkStyle = false
```

Comment:
Eww.  I mean, I see why this has to be here if you want to use it from the adaptive constructor, but it feels ugly to have a parameter to the adaptive constructor that is only in effect for one kind of rendering.  I suppose it's no different than `materialTapTargetSize` in that regard, though.

Would it be better to include "cupertino" in the name to make it clear that it doesn't affect Material radio buttons?  e.g. `useCupertinoCheckmarkStyle`?

Reply:
Yeah, I agree, I was trying different things to make it less gross, but this is where I landed.

Using Cupertino in the name instead of adaptive seems better to avoid possible confusion.